src/core/Akka.Streams/Serialization/StreamRefSerializer.cs

```diff
@@ -129,7 +127,7 @@ private SequencedOnNext DeserializeSequenceOnNext(byte[] bytes)
         {
             var onNext = Proto.Msg.SequencedOnNext.Parser.ParseFrom(bytes);
             var p = onNext.Payload;
-            var payload = _serialization.Deserialize(
+            var payload = system.Serialization.Deserialize(
```

Comment:
Actual fix. Instead of using a field assigned `Serialization`, we just retrieve it from `_system.Serialization` when we need it.